tests/page-objects/login/login.po.js

```diff
@@ -20,6 +21,17 @@ const changeLocale = locale => {
   return element(by.css(`.locale[name="${locale}"]`)).click();
 };
 
+const getLabelForUser = () => {
+  return element(by.css('#form > label:nth-child(2)'));
```

Comment:
Using the `nth-child` selector is ok but looking at the elements there is an attribute `for` that clearly defines the label. It would be less likely to break if the structure changes. 

`element(by.css('#form > label[for="user"]'));`